tests/helpers/mock_storage.go

```diff
@@ -134,6 +134,51 @@ func MustGetPostgresStorage(tb testing.TB, init bool) (storage.OCPRecommendation
 	}
 }
 
+// MustGetPostgresStorageDVO creates test postgres storage with credentials from config-devel for DVO storage
+func MustGetPostgresStorageDVO(tb testing.TB, init bool) (storage.DVORecommendationsStorage, func()) {
```

Comment:
this is a copy of `MustGetPostgresStorage`. Can't we create a `mustGetPostgresStorage` or similar that we can reuse in `MustGetPostgresStorage` and `MustGetPostgresStorageDVO`? I'm afraid we'll end up updating one of them and forgetting about the other

Reply:
Ok i will try to change it

Reply:
yep, that's what I was talking about yesterday @Jakub007d :)

Reply:
So after discussion with @Bee-lee we decided to push those changes and he will refactor the UTs when working on his task because i was getting some errors when tryi.ng to use inteface storage.Storage instead of strorage.OCPRecommendationsStorage